_trainings/2015-03-15-essential-essential-scala.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Essential Essential Scala
+title: Creative Scala
```

Comment:
These items were renamed...
I don't see any particular reason to retain the old names in the user facing content (I have not renamed the files).